react-native-reanimated.d.ts

```diff
@@ -445,6 +445,10 @@ declare module 'react-native-reanimated' {
       fn: (...args: A) => R
     ): (...args: Parameters<typeof fn>) => void;
     export function processColor(color: number | string): number;
+    export function createWorklet<A extends any[], R>(
+      fn: (...args: A) => R,
+      deps?: DependencyList
```

Comment:
Just realized I put deps here too
```suggestion
      fn: (...args: A) => R
```